gqcp/include/Basis/Integrals/BaseOneElectronIntegralBuffer.hpp

```diff
@@ -19,7 +19,6 @@
 
 
 #include "Mathematical/Representation/Matrix.hpp"
-
```

Comment:
Please set-up your VS Code to use the `clang-format` file for automatic formatting. The setup can be found on our website.